src/ludamus/templates/panel/facilitators.html

```
@@ -178,28 +178,30 @@
                                             </button>
                                         </form>
                                     {% endif %}
-                                    {% if facilitator.flagged_for_deletion %}
+                                    {% if facilitator.deleted_at %}
                                         <form method="post"
-                                              action="{% url 'panel:facilitator-unflag' slug=current_event.slug facilitator_slug=facilitator.slug %}">
+                                              action="{% url 'panel:facilitator-restore' slug=current_event.slug facilitator_slug=facilitator.slug %}">
                                             {% csrf_token %}
                                             <input type="hidden" name="next" value="{{ request.get_full_path }}">
                                             <button type="submit"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)